### PR TITLE
Add docs on using the debugger.

### PR DESCRIPTION
**Pull request by timholy**
I also have svg files of the two images. Should I commit those too?

### CODE REVIEW

File: docs/src/man/debugging.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  and the fact that the input string does not have any spaces at
+  positions higher than 7, this breakpoint would not have been hit.
+
+  To try out your new conditional breakpoint, first exit debugging by
+  clicking the "x" in the Debugger control bar.
+  Your console should
+  return to the normal `julia>` prompt, where you should enter
+  `Juno.@enter split("Hello there, world", isspace)`.
+  Click
+  "Continue" immediately: you'll return to the breakpoint, but now `j==13`
```
Comment (pfitzseb) [inline docs/src/man/debugging.md]: Maybe add a reference to `Juno.@run` here (as a shortcut for `@enter` -> Continue`)?

**Review by pfitzseb — COMMENTED** [review]
Thank you so much for writing this!

> I also have svg files of the two images. Should I commit those too?

Sure, why not. Makes updating the screenshots easier, and we don't really need to worry about the size of this repo.

**Comment by timholy** [comment]
OK, done.

**Comment by timholy** [comment]
It doesn't seem to have added to the hosted docs?

**Comment by pfitzseb** [comment]
Yeah, we're building on Julia 0.6 for some reason. Will fix.

**Pull request merged**